andela/mrm_api
# #164889269 Add check in time and meeting end time fields to Events model (V1)

### What does this PR do?
Adds check_in_time and meeting_end_time fields to Event model

### Description of the task to be completed?
Create new fields for capturing check-in time and time meeting ends.

### How should this be manually tested?
1. Clone the repo: `git clone https://github.com/andela/mrm_api.git`
2. Cd into the cloned repo 
3. Git pull branch ft-checkin-endmeeting-fields-events-v1-164889269
4. Run migrations
5. New table columns - `check_in_time` and `meeting_end_time`  should be added to the events table in the database

### Any background context you want to provide?
N/A

### What are the relevant pivotal tracker stories?
[#164889269](https://www.pivotaltracker.com/story/show/164889269)

### Checklist:
